nautobot_golden_config/migrations/0009_multiple_gc_settings.py

```diff
@@ -0,0 +1,137 @@
+# Generated by Django 3.1.14 on 2022-02-04 09:52
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def repo_convert_check_eligibility(apps, schema_editor):
+    """
+    Check if migration is applicable. Fail in case of many repositories created.
+    """
+    GoldenConfigSetting = apps.get_model("nautobot_golden_config", "GoldenConfigSetting")
+
+    settings_obj = GoldenConfigSetting.objects.first()
+
+    if settings_obj.backup_repository.all().count() > 1 or settings_obj.intended_repository.all().count() > 1:
+        raise ValueError("Please manually remove multiple repositories from Golden Config Settings before applying this migration")
+
+
+def convert_many_repos_part1(apps, schema_editor):
+    """
+    Add the current `backup_repository` and `intended_repository` objects values
```

Comment:
what is the difference between previous FK in backup/intended before and after? and why is different from templates?
I guess it's something related to the multipath, but I'm not getting it

Reply:
Previous are not FK, but Many 2 Many - in this case Django will delete all relations under m2m and will not move them under FK.